src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -157,6 +157,13 @@ In this message, the phrase "should not be reached" means that this error messag
     <value>Invalid value for `$(AndroidSequencePointsMode)`: {0}</value>
     <comment>The following are literal names and should not be translated: `$(AndroidSequencePointsMode)`</comment>
   </data>
+  <data name="XA0105" xml:space="preserve">
+    <value>The $(TargetFrameworkVersion) for {0} ({1}) is greater than the $(TargetFrameworkVersion) for the application project ({2}). Please increase the $(TargetFrameworkVersion) for the application project.</value>
```

Comment:
This makes the following wording change:

```diff
- The $(TargetFrameworkVersion) for {0} ({1}) is greater than the $(TargetFrameworkVersion) for your project ({2}). You need to increase the $(TargetFrameworkVersion) for your project.
+ The $(TargetFrameworkVersion) for {0} ({1}) is greater than the $(TargetFrameworkVersion) for the application project ({2}). Please increase the $(TargetFrameworkVersion) for the application project.
```

I did a quick local test to confirm that `<ResolveAssemblies/>` only runs for application projects, so I believe the change from "your project" to "the application project" is valid, but let me know if it looks problematic.